game/resource/English/ability/items/tooltip_mask_of_madness.txt

```diff
@@ -3,7 +3,7 @@
 //==================================================================================
 "DOTA_Tooltip_Ability_item_mask_of_madness_2"               "Mask of Madness"
 "DOTA_Tooltip_Ability_item_recipe_mask_of_madness_2"        "Mask of Madness Recipe"
-"DOTA_Tooltip_Ability_item_mask_of_madness_2_Description"                           "<h1>Active: Berserk</h1>Gives %berserk_bonus_attack_speed% attack speed and %berserk_bonus_movement_speed%%% movement speed, but reduces your armor by %berserk_armor_reduction% and silences you.  Lasts %berserk_duration% seconds.\n<h1>Passive: Lifesteal</h1>Heals the attacker for %lifesteal_percent%%% of attack damage dealt. <br><br>Mask of Madness is a Unique Attack Modifier, and does not stack with other Unique Attack Modifiers."
+"DOTA_Tooltip_Ability_item_mask_of_madness_2_Description"                           "<h1>Active: Berserk</h1>Gives %berserk_bonus_attack_speed% attack speed and %berserk_bonus_movement_speed%%% movement speed, but reduces your armor by %berserk_armor_reduction% and silences you.  Lasts %berserk_duration% seconds.\n<h1>Passive: Lifesteal</h1>Heals the attacker for %lifesteal_percent%%% of attack damage dealt."
```

Comment:
There's an extra space before "Lasts". It should be `<h1>Active: Berserk</h1>Gives %berserk_bonus_attack_speed% attack speed and %berserk_bonus_movement_speed%%% movement speed, but reduces your armor by %berserk_armor_reduction% and silences you.  Lasts %berserk_duration% seconds.\n<h1>Passive: Lifesteal</h1>Heals the attacker for %lifesteal_percent%%% of attack damage dealt.`